BeeKit/HeathKit/HealthKitMetric.swift

```diff
@@ -12,6 +12,7 @@ public enum HealthKitCategory : String, CaseIterable {
     case Heart = "Heart"
     case Mindfulness = "Mindfulness"
     case Nutrition = "Nutrition"
+    case SelfCare = "Self Care"
```

Comment:
These categories map to the categorizations in the Health app. Toothbrushing is under "Other".